core/domain/wipeout_service.py

```diff
@@ -267,31 +339,58 @@ def _hard_delete_explorations_and_collections(pending_deletion_request):
         pending_deletion_request.user_id,
         pending_deletion_request.exploration_ids,
         force_deletion=True)
+    pending_deletion_request.exploration_ids = []
+    save_pending_deletion_requests([pending_deletion_request])
     collection_services.delete_collections(
         pending_deletion_request.user_id,
         pending_deletion_request.collection_ids,
         force_deletion=True)
+    pending_deletion_request.collection_ids = []
+    save_pending_deletion_requests([pending_deletion_request])
 
 
-def _generate_activity_to_pseudonymized_ids_mapping(activity_ids):
-    """Generate mapping from activity IDs to pseudonymous user IDs.
+def _generate_entity_to_pseudonymized_ids_mapping(entity_ids):
+    """Generate mapping from entity IDs to pseudonymous user IDs.
 
     Args:
-        activity_ids: list(str). List of activity IDs for which to generate
-            new pseudonymous user IDs. The IDs are of activities that were
-            modified in some way by the user who is currently being deleted.
+        entity_ids: list(str). List of entity IDs for which to generate new
+            pseudonymous user IDs. The IDs are of activities that were modified
```

Comment:
Can you be more specific in this docstring about what exactly "activities" means here?